examples/liquid.ini

```diff
@@ -190,6 +190,7 @@ two_factor_auth = false
 ;dokuwiki = on
 ;codimd = on
 ;wikijs = on
+;bbb = off
```

Comment:
see comment below about making BBB work like grist & prophecies (default off, because requires manual config) -- also move this line to the place below after grist & prophecies